physionet-django/notification/utility.py

```diff
@@ -374,17 +374,21 @@ def mailto_supervisor(request, application):
     return mailto
 
 
-def mailto_process_credential_complete(request, application):
+def mailto_process_credential_complete(request, application, include=True):
```

Comment:
The purpose of the `include` argument isn't clear here. Might be helpful to:
- describe the argument/s in the docstring (styled like https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)
- rename `include` to something that more clearly describes its role.

Reply:
The include is to include the applicant comments in the email mailto function.

Reply:
Looks like it also controls whether or not "Dear NAME" is added to the start of the email template?

Reply:
That is correct, KP has a pre-made email in which he copy pastes there. So One is for acceptance, and the other is for communication. Because of that pre-made email text that he copy pastes, I thought it was easier to just leave it all blank and allow him to be him

Reply:
In that case, perhaps better to have as two separate arguments? e.g. something like:

`salutation=True, comments=True`

Reply:
I changed include to comments, but salutation is always there, so there is no need to create a extra variable.